manifests/service.pp

```diff
@@ -49,4 +49,66 @@
                   ],
   }
 
+  # if the Sentry config changes, do a full restart of the Sentry Celery workers
+  exec { 'restart-sentry-worker':
+    command     => '/usr/bin/systemctl stop sentry-worker; /usr/bin/systemctl start sentry-worker',
+    path        => '/bin:/usr/bin',
+    refreshonly => true,
+    subscribe   => Class['::sentry::config'],
+  }
+
+  # Sentry Celery Beat
+  file { '/etc/systemd/system/sentry-beat.service':
+    ensure  => present,
+    mode    => '0644',
+    owner   => 'root',
+    group   => 'root',
+    content => template('sentry/sentry-beat.service.erb'),
+    notify  => Exec['enable-sentry-services'],
+  }
+
+  service { 'sentry-beat':
```

Comment:
Great catch!

The [official Sentry documentation](https://docs.getsentry.com/on-premise/server/queue/) references running each process separately, and makes no mention of using the `-B` flag.  I honestly don't know the ramifications of either options.

Any preferences?  I can yank out the explicit beat stuff from this PR, or I can remove the `-B` flag from the worker invocation.